tests/test_dataset.py

```diff
@@ -420,11 +483,31 @@ def test_graph_standardization_logic(self):
             GraphDataset(
                 hdf5_path = hdf5_path,
                 target='binary',
-                standardize=True,
                 train=False,
                 dataset_train=dataset_train
             )
+            
+    def test_feature_transform_mean_std(self):
+        node_feat_test='bsa'
+        hdf5_path = "tests/data/hdf5/train.hdf5"
+        features_transform={'bsa':{'transform':lambda t:np.log(t+1),'standardize':True}}
+        
+        #calculate transformation and standardization form GraphDataset object
+        dataset_test_transform = GraphDataset(
+            hdf5_path = hdf5_path,
+            target='binary',
+            node_features = [node_feat_test],
+            edge_features =[],
```

Comment:
From the document of GraphDataSet, we cannot pass the edge_features as None

Reply:
From the docs, it is an [Optional parameter](https://docs.python.org/3/library/typing.html#:~:text=type%20expressions.-,typing.Optional%C2%B6,-Optional%20type.), which means that it can be None :)